platform-coop-toolkit/pcc-framework
# Add support for People page

## Description

- [x] Remove archive page support from events and people
- [x] Remove topics in lieu of tags
- [x] Add column to people table to show People roles

## Steps to test

1. Verify that people can be tagged.
2. Verify that `/events` and `/people` redirect to the home page.

## Additional information

Not applicable.

## Related issues

- https://github.com/platform-coop-toolkit/pcc/issues/84
